### PR TITLE
Feature/404 page

### PR DESCRIPTION
**Pull request by KorolyovaLara**

### CODE REVIEW

File: routes/api/gameApi.js [hunk 1]
```diff
@@ -19,13 +37,13 @@ router.post("/", withAuth, async (req, res) => {
       });
 
       res.json(game);
-    } catch (e) {
-      console.log(e);
+    } catch (err) {
+      console.log(err);
```
Comment (strahinjaajvaz) [inline routes/api/gameApi.js:28]: remove

**Review by strahinjaajvaz — COMMENTED** [review]
Can you please add a screen shot of what you've built in regards to the ui

File: routes/auth/auth.js [hunk 2]
```diff
@@ -62,6 +62,7 @@ router.post("/register", async (req, res) => {
       },
     });
   } catch (e) {
+    //console.log(e);
```
Comment (strahinjaajvaz) [inline routes/auth/auth.js]: remove.

File: server.js [hunk 3]
```diff
@@ -25,6 +25,11 @@ app.use(express.static(path.join(__dirname, "public")));
 
 app.use(routes);
 app.use(setNewToken);
+// landing page and others
```
Comment (strahinjaajvaz) [inline server.js]: wrong line. 
this comment should be on line 25....

File: routes/api/gameApi.js (hunk 1 above)
Comment (KorolyovaLara) [reply routes/api/gameApi.js:28]: I will remove that fully as belong to different branch

File: routes/api/gameApi.js [hunk 4]
```diff
@@ -1,11 +1,29 @@
...
+const Users = require("../../models/Users");
+const GameCollection = require("../../models/GameCollection");
 const { withAuth } = require("../../utils/auth");
 
 // find all games for the user
-router.get("/", withAuth, async (req, res) => {});
+router.get("/id", withAuth, async (req, res) => {
+  try {
+    const userGames = await Games.findByPk(req.params.id, {
+      include: [{ model: Users, through: GameCollection, as: "user_games" }],
```
Comment (strahinjaajvaz) [inline routes/api/gameApi.js]: have you tested this?

Comment (KorolyovaLara) [reply routes/api/gameApi.js]: removing from this branch as i am working on it in a different one now

**Comment by KorolyovaLara** [comment]
@strahinjaajvaz all small "issues" are removed

**Comment by KorolyovaLara** [comment]
Screen Shot 2021-06-07 at 8 34 57 pm

**Pull request merged**